Makefile

```diff
@@ -54,9 +54,10 @@ on-dup:
 fmt: groupimports
 	go fmt ./...
 
-mod:
+tidy:
+	@echo "go mod tidy"
 	GO111MODULE=on go mod tidy
-	@git diff --exit-code -- go.sum go.mod
+	@git diff --exit-code -- go.mod
```

Comment:
I'm not sure the reason it cannot pass in travis CI. I delete go.sum and run `make tidy` to generate go.sum but hasn't any difference between previous one.